File: CJY/BJ/No_10250.js
const rl = require("readline").createInterface({input: process.stdin, output: process.stdout});
let T = 0;
let answer = "";

rl.on("line", (s) => {
    if(T > 0) {
        const arr = s.split(" ");
        let H = parseInt(arr[0]);
        let N = parseInt(arr[2]);
        let a = N % H === 0 ? H : ~~(N % H);
        let b = a === H ? ~~(N / H) : ~~(N / H) + 1;
        answer += a + (b > 9 ? "" : "0") + b + "\n";
        T--;
        if(T === 0) {
            console.log(answer);
            rl.close();
        }
    } else {
        T = parseInt(s);
    }
}).on("close", () => {process.exit();});

File: CJY/BJ/No_10250.java
import java.io.*;

public class No_10250 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int T = Integer.parseInt(br.readLine());
        BufferedWriter bw = new BufferedWriter(new OutputStreamWriter(System.out));
        for (int i = 0; i < T; i++) {
            String[] arr = br.readLine().split(" ");
            int H = Integer.parseInt(arr[0]);
            int N = Integer.parseInt(arr[2]);
            int a = N % H == 0 ? H : N % H;
            int b = a == H ? N / H : (N / H) + 1;
            bw.write(a + (b > 9 ? "" : "0") + b);
            bw.newLine();
        }
        br.close();
        bw.flush();
        bw.close();
    }
}